sqldelight-gradle-plugin/src/test/integration-android-variants/src/release/sqldelight/com/squareup/sqldelight/integration/Release.sq

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE release_table (
+  value TEXT NOT NULL
+);
+
+INSERT INTO release_table VALUES('release');
+
+selectAll:
+SELECT *
+FROM release_table;
```

Comment:
Do you want this test to include referencing src/main tables from variant queries? In other words, should you join `group` in here and `Debug.sq`?

Reply:
nice i like that :+1: